manager/src/lib.rs

```diff
@@ -170,12 +171,34 @@ impl CronManager {
         self.slots.get(&current_slot).unwrap_or(default_vec)
     }
 
+    /// Most useful for debugging at this point.
+    pub fn get_all_tasks(&self, slot: Option<U128>) -> Vec<Base64VecU8> {
+        let mut ret: Vec<Base64VecU8> = Vec::new();
+        if let Some(slot_number) = slot {
+            // User specified a slot number, only return tasks in there.
+            let tasks_in_slot = self.slots.get(&slot_number.0).expect("Couldn't find tasks for given slot.");
+            for task in tasks_in_slot.iter() {
+                ret.push(Base64VecU8::from(task.to_vec()));
+            }
+        } else {
+            // Return all slots
+            for slot in self.slots.iter() {
```

Comment:
@mikedotexe not a bad idea, i guess i figured not to bother as it might be a lot of data. I guess we'll have to see :)

Reply:
Yeah I think this whole function is good for debugging in the beginning